src/lib/sfge/SfgeWrapper.ts

```diff
@@ -142,7 +146,7 @@ export class SfgeWrapper extends CommandLineSupport {
 		const sourceListFile = await this.createInputFile(this.projectDirs);
```

Comment:
While QA'g this as an internal implementation, we had a discussion around passing a single file with both target and source instead of separate files but didn't have an iteration to make the changes. Could you address it with these changes? A single input file would help us move towards a configuration of sorts for executing SFGE.